modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/actions/ActionsDropdown.js

```diff
@@ -127,7 +127,7 @@ function ActionsDropdown({
 		return null;
 	}
 
-	if (!inlineEditingAlwaysOn && actions.length === 1) {
+	if (!inlineEditingAlwaysOn && !context.forceActionsInDropdown && actions.length === 1) {
```

Comment:
LGTM!

I don't think there is a way we can automatically detect _possibility_ of different action counts. It would have to be something that would parse code of `FDSActionProvider`s. It has to be a something set by a human, in a flag like this.

Reply:
About naming:
*  using "force" looks to me a bit strong. Perhaps "use" or "render"
* Also, flag talks about actions when what it's doing is to use a dropdown when there is a single action. 
 
So the thing this flag is doing looks better described if using something like `useDropdownForSingleAction` or, shorter `singleActionDropdown`

Reply:
Moving this to https://github.com/liferay-frontend/liferay-portal/pull/2992